.github/workflows/tests.yml

```diff
@@ -34,14 +34,116 @@ jobs:
       env:
         GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
-        . asdf/asdf.sh
+        . $PWD/asdf/asdf.sh
         asdf plugin-test java "$GITHUB_WORKSPACE" --asdf-plugin-gitref "$GITHUB_SHA" --asdf-tool-version adoptopenjdk-8.0.252+9.1.openj9-0.20.0 java -version
     - name: macOS Check java_home integration
+      if: matrix.os == 'macOS-latest'
       env:
         GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
         export ASDF_CONFIG_FILE=${HOME}"/.asdfrc"
         echo "java_macos_integration_enable = yes" > "${ASDF_CONFIG_FILE}"
-        . asdf/asdf.sh
+        . $PWD/asdf/asdf.sh
         asdf plugin-test java "$GITHUB_WORKSPACE" --asdf-plugin-gitref "$GITHUB_SHA" --asdf-tool-version zulu-8.52.0.23 /usr/libexec/java_home -V 2>&1 | grep "Zulu 8.52.0.23"
+    - name: Setup JAVA_HOME setting
+      env:
+        GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        export ASDF_CONFIG_FILE=${HOME}"/.asdfrc"
+        echo "java_macos_integration_enable = yes" > "${ASDF_CONFIG_FILE}"
+        . $PWD/asdf/asdf.sh
+        asdf plugin add java "$GITHUB_WORKSPACE"
+        asdf install java adoptopenjdk-8.0.252+9.1.openj9-0.20.0
```

Comment:
After the changes in #140, this (and all subsequent usages of this identifier in this file) should probably be `adoptopenjdk-openj9-8.0.252+9.1.openj9-0.20.0` or `adoptopenjdk-8.0.252+9.1`.

Reply:
Will see what is needed now to get things to work again properly. Been a long time since I last worked on this...